acs-integration-tests/src/test/java/com/ge/predix/cloudfoundry/client/CloudFoundryApplicationHelper.java

```diff
@@ -217,6 +226,22 @@ public void bindServiceInstances(final String applicationName, final List<String
                         .build()));
     }
 
+    public String getUnusedServiceInstance(final List<String> instanceNames) {
+
+        for (String instance : instanceNames) {
+            try {
+                if (this.cloudFoundryOperations.services().getInstance(GetServiceInstanceRequest.builder()
+                        .name(instance).build()).block().getApplications().size() == 0) {
+                    return instance;
+                }
+            } catch (RuntimeException e) {
+                LOGGER.info(String.format("Instance '%s' does not exist", instance), e);
```

Comment:
I don't think info has a signature with params and throwable 
https://www.slf4j.org/apidocs/org/slf4j/Logger.html